technique/reflectometry/script_2_OFFSPEC.py

```diff
@@ -0,0 +1,142 @@
+# ## This script was generated on 15/02/2022, at 16:55:10
+# ## with ScriptMaker (c) Maximilian Skoda 2020 
+# ## Enjoy and use at your own risk.
+import time
+
+from technique.reflectometry import *  # __all__ defined in __init__.py
+import logging
+import sys, io
+import os
+
+try:
+    # pylint: disable=import-error
+    from genie_python import genie as g
+except ImportError:
+    from .technique.reflectometry.mocks import g
+
+# Remove all handlers associated with the root logger object.
+for handler in logging.root.handlers[:]:
+    logging.root.removeHandler(handler)
+
+# Set up new logger
+# logging.basicConfig(
+    # handlers=[
+        # logging.StreamHandler(sys.stdout),
+        # logging.FileHandler("example.log", mode="w"),
+    # ],
+    # level=logging.DEBUG,
+    # format="%(asctime)s | %(levelname)s | %(message)s",     # %(name)s -
+    # datefmt="%Y-%m-%d %H:%M:%S",
+# )
```

Comment:
Just a note (I know some of this is commented out), but this `basicConfig` call or other similar calls elsewhere in the codebase may cause other libraries to start printing their debug messages to screen, not just this script. Removing all root handlers may also have unintended side-effects.

It is likely better to add & remove handlers to just the logger(s) you're interested in, rather than the global logger(s).

This can then get worrying for users as they start seeing lots of red text in the console that they don't care about, from all sorts of libraries which IBEX uses in the background (e.g. matplotlib, channel access libraries, `bluesky` in future, etc etc)